### PR TITLE
Add Windows Application Package properties to Microsoft.Build.CommonTypes.xsd

### PR DESCRIPTION
**Pull request by XianghanWang**
Add two properties to fix warnings in DesktopBridge project file.

### CODE REVIEW

**Comment by XianghanWang** [comment]
@AndyGerlicher  @rainersigwald could you help to review it? Thanks.

File: src/MSBuild/Microsoft.Build.CommonTypes.xsd [hunk 1]
```diff
@@ -4308,6 +4308,22 @@ elementFormDefault="qualified">
...
+            <xs:documentation>
+                <!-- _locID_text="WapProjPath" _locComment="" -->Windows Application Packaging project-specific: Path to Windows Application Packaging project root folder.
+            </xs:documentation>
+        </xs:annotation>
+    </xs:element>
+
+    <xs:element name="EntryPointProjectUniqueName" type="msb:StringPropertyType" substitutionGroup="msb:Property">
+        <xs:annotation>
+            <xs:documentation>
+                <!-- _locID_text="EntryPointProjectUniqueName" _locComment="" -->Windows Application Packaging project-specific: Relative path to entry point project file
```
Comment (rainersigwald) [inline src/MSBuild/Microsoft.Build.CommonTypes.xsd]: Nit to match others:
```suggestion
                <!-- _locID_text="EntryPointProjectUniqueName" _locComment="" -->Windows Application Packaging project-specific: Relative path to entry point project file.
```

Comment (XianghanWang) [reply src/MSBuild/Microsoft.Build.CommonTypes.xsd]: Thanks Rainer. 

It is even better if we can make it for 16.2.

I will create a new PR for 16.2 branch.

Comment (XianghanWang) [reply src/MSBuild/Microsoft.Build.CommonTypes.xsd]: @rainersigwald, could you help review the new PR in vs16.2? Thanks.
https://github.com/microsoft/msbuild/pull/4444

Comment (XianghanWang) [reply src/MSBuild/Microsoft.Build.CommonTypes.xsd]: Per  livarcocc's comment, close 4444 and use this PR

**Pull request merged**